index.html

```diff
@@ -1845,13 +1845,14 @@ <h4>Semantic search: SPARQL</h4>
                             The support for SPARQL search using HTTP <code>POST</code> method
                             at `/search/sparql` endpoint is OPTIONAL.
                         </span>
-                        <code>UPDATE</code> queries are out of the scope for the API.
+			SPARQL <code>UPDATE</code> queries are disallowed.
```

Comment:
Probably should be an assertion, and should be a statement about the content of the query itself.  Maybe:
"The SPARQL query MUST NOT include <code>UPDATE</code> statements."  Or it can be dealt with in a different way below (by stating what IS allowed, "a request with a query different".).  We could also add "In particular, <code>UPDATE</code> is disallowed."